### PR TITLE
Add Basics -> Wallet -> Getting started

### PR DESCRIPTION
**Pull request by 5A11**
This PR adds Basics --> Wallet --> Getting started page.

### CODE REVIEW

**Comment by StephenBrownFetch** [comment]
@5A11 Looks good overall but I think we might not need all 4 danger boxes when they largely say the same thing which is effectively "Look after your mnemonic seed or you will lose access to your assets, or someone could steal them" 

Having all 4 makes the page a bit   s t r e s s f u l   so I'd recommend reducing to 1 or 2 if possible

Is there a reason one info box is named "Note" and another info box named "Info", would recommend sticking to Info for consistency if possible

**Comment by 5A11** [comment]
> @5A11 Looks good overall but I think we might not need all 4 danger boxes when they largely say the same thing which is effectively "Look after your mnemonic seed or you will lose access to your assets, or someone could steal them"
> 
> Having all 4 makes the page a bit s t r e s s f u l so I'd recommend reducing to 1 or 2 if possible
> 
> Is there a reason one info box is named "Note" and another info box named "Info", would recommend sticking to Info for consistency if possible

We can name the boxes anything really. The first one was more of a disclaimer/warning, hence I used the info design (to reduce the orange/red as you brought up above) but used Note to stress importance.

I kinda agree with your comment that there is too much red/orange. The idea however is that a reader should mostly read only one of the headers depending on their use case. So they shouldn't really refer to the red/oranges in the other sections. For the same reason, if we omit the warnings under any of the headers people referring only to that section would not see the warning.

The supported boxes are here https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types. 
Feel free to have a look and suggest alternatives given the above.

**Comment by StephenBrownFetch** [comment]
@5A11 Okay, understood as to why there is one per section. I'd suggest to keep both the blue titled as "Note", and change the Danger to Warning. and where we have two under "To Create a New Account" we can consolidate to one. I will try to make the updates into the code on GitHub

File: docs/basics/wallet/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Compatibility
+
+The Fetch.ai Network Wallet works on all Chromium-based web browsers, including [Chrome](https://www.google.com/chrome/), [Brave](https://brave.com), [Edge](https://www.microsoft.com/edge) and [Decentr](https://decentr.net).
+
+## Get the Wallet
+
+Get the Fetch.ai Network Wallet from the [Chrome web store](https://chrome.google.com/webstore/detail/fetchai-network-wallet/ellkdbaphhldpeajbepobaecooaoafpg/related?hl=en-GB).
+
+!!! info "Note"
```
Comment (StephenBrownFetch) [inline docs/basics/wallet/getting_started.md]: !!! info "Note" -> !!! info "Info"

File: docs/basics/wallet/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+When you open the Fetch wallet for the first time, you can create a new account on the Fetch network:
+
+* Select `Create new account`
+* Backup your mnemonic seed securely.
+
+!!! danger
+    Keep it safe! Anyone with your mnemonic seed can access your wallet and take your assets.
+
+!!! danger
```
Comment (StephenBrownFetch) [inline docs/basics/wallet/getting_started.md]: can remove

File: docs/basics/wallet/getting_started.md (hunk 1 above)
Comment (StephenBrownFetch) [inline docs/basics/wallet/getting_started.md]: !!! info "Note" -> !!! info "Info"

File: docs/basics/wallet/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## To Create a New Account
+
+When you open the Fetch wallet for the first time, you can create a new account on the Fetch network:
+
+* Select `Create new account`
+* Backup your mnemonic seed securely.
+
+!!! danger
+    Keep it safe! Anyone with your mnemonic seed can access your wallet and take your assets.
```
Comment (StephenBrownFetch) [inline docs/basics/wallet/getting_started.md]: can add "Lost mnemonic seed cannot be recovered! If you lose your mnemonic seed you will lose access to your wallet."

File: docs/basics/wallet/getting_started.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    At this time, you cannot run the [Keplr browser wallet](https://chrome.google.com/webstore/detail/keplr/dmkamcknogkgcdfhhbddcghachkejeap?hl=en) and Fetch Network browser wallet together, as they interfere. Please disable the Keplr wallet in extensions before using the Fetch wallet.
+
+## To Create a New Account
+
+When you open the Fetch wallet for the first time, you can create a new account on the Fetch network:
+
+* Select `Create new account`
+* Backup your mnemonic seed securely.
+
+!!! danger
```
Comment (StephenBrownFetch) [inline docs/basics/wallet/getting_started.md]: warning

File: docs/basics/wallet/getting_started.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+## Existing Account
+
+### Import an existing account
+
+If you have an account on the Fetch network, for example having had one already on the Fetch wallet and want to access it again, or have an account on another wallet (e.g. Cosmostation, Keplr, ...) and wish to bring it to the Fetch wallet for the first time:
+
+* Select `Import existing account`
+* Enter your mnemonic seed or private key.
+
+!!! danger
```
Comment (StephenBrownFetch) [inline docs/basics/wallet/getting_started.md]: warning

File: docs/basics/wallet/getting_started.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+### Use ledger hardware wallet
+
+If you have a Ledger hardware wallet and wish to keep your key and mnemonics on that device while using the Fetch wallet:
+
+* Select `Import ledger`
+* Give your account a name and set a password. This password will be used the next time you want to use the wallet or make important changes to your account.
+* Follow the on-screen instructions.
+
+!!! danger
```
Comment (StephenBrownFetch) [inline docs/basics/wallet/getting_started.md]: warning

**Review by StephenBrownFetch — DISMISSED** [review]

File: docs/basics/wallet/getting_started.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
+The Fetch.ai Network Wallet allows you to interact with the Fetch network through your browser.
```
Comment (VijaySharma815) [inline docs/basics/wallet/getting_started.md:1]: Fetch Wallet

File: docs/basics/wallet/getting_started.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+[comment]: <> (There are many other wallets you can use. The Fetch.ai Network Wallet is a non-custodial browser-extension wallet that is maintained by the Fetch.ai team. )
+
+## Compatibility
+
+The Fetch.ai Network Wallet works on all Chromium-based web browsers, including [Chrome](https://www.google.com/chrome/), [Brave](https://brave.com), [Edge](https://www.microsoft.com/edge) and [Decentr](https://decentr.net).
+
+## Get the Wallet
+
+Get the Fetch.ai Network Wallet from the [Chrome web store](https://chrome.google.com/webstore/detail/fetchai-network-wallet/ellkdbaphhldpeajbepobaecooaoafpg/related?hl=en-GB).
```
Comment (VijaySharma815) [inline docs/basics/wallet/getting_started.md:11]: Fetch Wallet

File: docs/basics/wallet/getting_started.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+!!! info
+    At this time, you cannot run the [Keplr browser wallet](https://chrome.google.com/webstore/detail/keplr/dmkamcknogkgcdfhhbddcghachkejeap?hl=en) and Fetch Network browser wallet together, as they interfere. Please disable the Keplr wallet in extensions before using the Fetch wallet.
```
Comment (VijaySharma815) [inline docs/basics/wallet/getting_started.md:14]: Fetch Wallet

**Review by VijaySharma815 — APPROVED** [review]
LGTM

**Pull request merged**